jax/_src/scipy/test_eigh.py

```diff
@@ -0,0 +1,49 @@
+"""Test for eigh.py."""
+from jax._src.scipy import eigh
+
+import jax.numpy as jnp
+from jax import lax
+import numpy as np
+import pytest
+
+Ns = [16, 256]
+precisions = [lax.Precision.HIGHEST,]
+
+@pytest.mark.parametrize("N", Ns)
```

Comment:
I'd avoid using pytest marks. We don't always use pytest to run tests (e.g., in our internal CI), so the tests need to work without pytest as the test runner, which means you can't use marks.

The test should work if you just run `python test_eigh.py` without pytest; you'd probably need to add a `__main__`  and move the tests into a class for this to work.

Typically we use absl's `parameterized` module for parametric tests.

Take a look at, say, `lax_numpy_test.py` for some examples.

Reply:
Done